dns/bind/src/opnsense/mvc/app/models/OPNsense/Bind/Dnsbl.xml

```diff
@@ -0,0 +1,20 @@
+<model>
+    <mount>//OPNsense/bind/dnsbl</mount>
+    <description>DNSBL configuration</description>
+    <version>1.0.0</version>
+    <items>
+        <enabled type="BooleanField">
+            <default>0</default>
+            <Required>Y</Required>
+        </enabled>
+        <type type="OptionField">
+            <default>all</default>
+            <Required>N</Required>
+            <OptionValues>
+                <ad>ADWare Lists</ad>
+                <mw>Malware Lists</mw>
+                <all>All in on</all>
```

Comment:
multiselect is better than explicit all value

Reply:
There's a general problem with the lists. It's possible that a domain is in two different lists and I have no idea how to "uniq" over multiple files. That's why I cat them together. Most of the scripts around have such logics to avoid duplicate zone files (which would crash the daemon)

Reply:
can deal with this later, not a problem. can help script this :)